src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/With Organisation Context/S-105.18/S-105.18.td.json

```diff
@@ -0,0 +1,37 @@
+{
+  "title": "Must increment Assigned User Count when assigning a user and case role for a specific case if there was only [CREATOR] case user role assignment with the respective values in the request (by a user calling through/from an authorised application)",
+
+  "_guid_": "S-105.18",
+  "_extends_": "F-105_Add_Case_Assigned_User_Roles_Base",
+
+  "specs": [
+    "Richard - who can create a case",
+    "Dil - who is to add some case role assignment for a case",
+    "Olawale - with an active solicitor profile and valid User ID",
+    "is made from an authorised application, by Dil, with the Case ID of C1, User ID of Olawale, proper Case Role CR-2 and the Organisation ID of Olawale"
+  ],
+
+  "users": {
+    "_extends_": "F-105_Users"
```

Comment:
put here the user Dil, to make it consistent with the injected token in the request header

Reply:
Corrected